src/components/forms/DocumentUploadForm/DocumentUploadProgress.vue

```diff
@@ -13,6 +13,8 @@
 <script>
 import VProgressCircle from '@endpass/ui/kit/VProgressCircle';
 
+const MIN_PROGRESS_VALUE = 2;
```

Comment:
maybe, move that logic to the store?

Reply:
this logic for view only, store must always return correct value